specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/examples/HDI_SparkBatch_List.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "endpoint": "clustername.azurehdinsight.net",
+    "from": 0,
+    "size": 2,
+    "detailed": true
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "from": 0,
+        "total": 2,
+        "sessions": []
```

Comment:
Could we get some SparkBatchJob example so the CI can actually check the format of this operation?

Reply:
I will make all examples  complete as soon as possible.